canonical-web-and-design/ubuntu.com
# WIP: Straighten quotes and then smarten them with js

## Done

- Straighten quotes and then smarten them with js
- NOTE: there might be better ways to include the [js](https://smartquotes.js.org/), and it isn't passing all the lint checks

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the straighted quotes are curly
